tests/tests_pytorch/utilities/data/test_dataset.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from lightning.pytorch.utilities.data.dataset import LightningDataset
+from lightning.pytorch.utilities.data.fileio import OpenCloudFileObj
+
+
+def test_lightning_dataset(tmpdir):
```

Comment:
These tests require an internet connection, right?
if so pls add skip if you are offline or the target bucket is not reachable...

Reply:
added